### ISSUE TITLE
[FR] - Add a synchronisation percentage figure to the query tip command output

### ISSUE THREAD

**Issue by erikd** [input-output-hk/cardano-node#2569]
**Internal**

**Area**
CLI

**Describe the feature you'd like**
The `cardano-cli query tip` command output currently looks like:
```
{
    "epoch": 238,
    "hash": "aafe6262fcccabf3b7b6603cee0a7d8ff91f89e90ecf5444e98fed20500d65e0",
    "slot": 17818160,
    "block": 5146835
}
```
It would be nice if the synchronization percentage could be included.

**Describe alternatives you've considered**
There is [a script](https://forum.cardano.org/t/check-sync-status-of-node/36500/12), but the script has been confirmed to be wrong.

**Comment by erikd** [comment]
The ticket #2559 was raised by someone using the script (originally written for a testnet) on mainnet.

**Comment by erikd** [comment]
#2568 is related and should be tackled at the same time as this one.

**Comment by dorin100** [comment]
this is fixed starting with node 1.29.0
```
{
    "epoch": 302,
    "hash": "e956ec99f769479d75de0f0fd62d1ca051f2cb4768a4485f116e75fb845e45b7",
    "slot": 45487490,
    "block": 6508194,
    "era": "Alonzo",
    "syncProgress": "100.00"
}
```
